api/src/main/java/org/openmrs/api/context/Daemon.java

```diff
@@ -103,6 +103,18 @@ public void run() {
 		return (Module) startModuleThread.returnedObject;
 	}
 	
+	public static String getDaemonUserUuid() {
+		return DAEMON_USER_UUID;
+	}
+
+	public static ThreadLocal<Boolean> getIsdaemonthread() {
+		return isDaemonThread;
+	}
+
+	public static ThreadLocal<User> getDaemonthreaduser() {
```

Comment:
Can you fix the above method name as per? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions